grails-app/conf/Config.groovy

```diff
@@ -1081,6 +1081,7 @@ openboxes.identifier.productCode.generatorType = IdentifierGeneratorTypeCode.SEQ
 openboxes.identifier.productCode.delimiter = Constants.DEFAULT_IDENTIFIER_SEPARATOR
 openboxes.identifier.productCode.format = "\${productTypeCode}\${delimiter}\${sequenceNumber}"
 openboxes.identifier.productCode.properties = ["productTypeCode": "code"]
+openboxes.identifier.defaultProductType.id = "DEFAULT"
```

Comment:
We should use the code property here, right?

Reply:
No. Code is nullable.

Reply:
I'm saying that code should be non nullable and unique. If we can't do it now it should be added as a future improvement.